salt/data-pipeline/nifi-registry.sls

```diff
@@ -4,7 +4,7 @@
 download-nifi-registry:
     file.managed:
         - name: /root/downloads/nifi-registry-0.4.0-bin.tar.gz
-        - source: https://www-eu.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
+        - source: https://archive.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
```

Comment:
why it has been removed from the other mirror? Will it happen again on `archive.apache.org`?

Reply:
`https://www-eu.apache.org/dist/` seems to be a backup mirror only:
> Please only use the backup mirrors to download KEYS, PGP signatures and hashes (SHA* etc) -- or if no other mirrors are working. -- https://www.apache.org/dyn/closer.cgi

Reply:
https://archive.apache.org/dist/nifi/nifi-registry/ seems to have all the possible versions so I'm cool with it.

Reply:
(https://apache.org/dist/nifi/nifi-registry/ would be bad instead)